config/database.yml

```diff
@@ -0,0 +1,25 @@
+# SQLite. Versions 3.8.0 and up are supported.
+#   gem install sqlite3
+#
+#   Ensure the SQLite 3 gem is defined in your Gemfile
+#   gem "sqlite3"
+#
+default: &default
+  adapter: sqlite3
+  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+  timeout: 5000
```

Comment:
- [ ] @jodavid444 kindly consider  changing your database from SQLite to Postgresql, this is a project requirement as shown below
![image](https://user-images.githubusercontent.com/61408860/236012079-f2533b7a-ecb4-4eb1-b0ce-05af3f692cbf.png)

Reply:
Thank you for spending quality time reviewing my project, I have updated the database.yml file meeting up to project requirement, kindly review it again @danmainah